src/frontend/src/index.ts

```diff
@@ -103,17 +106,14 @@ const init = async () => {
     // Register this device (tentatively)
     await registerTentativeDevice(addDeviceAnchor, connection);
 
+    const i18n = new I18n();
+    const staticCopy = i18n.staticLang(copyJson);
+
     // Show a good bye message
-    const container = document.getElementById("pageContent") as HTMLElement;
-    render(
-      html`<h1
-        style="position: absolute; max-width: 100%; top: 50%; transform: translate(0, -50%);"
-        data-role="notify-auth-success"
-      >
-        You may close this page.
-      </h1>`,
-      container
-    );
+    showMessage({
+      message: staticCopy.close_page_device_added,
+      role: "notify-device-added",
```

Comment:
renamed to `notify-device-added` on purpose right?

Reply:
yep! thanks for paying attention though